wallet/tests/output_preparation.rs

```diff
@@ -370,6 +370,52 @@ async fn output_preparation() -> Result<()> {
     // address, sdr, expiration
     assert_eq!(output.unlock_conditions().unwrap().len(), 3);
 
+    let output = account
+        .prepare_output(
+            OutputOptions {
+                recipient_address: recipient_address.clone(),
+                amount: 8001,
+                assets: None,
+                features: None,
+                unlocks: None,
+                storage_deposit: None,
```

Comment:
What if there is already an sdr? Maybe add a test?

Reply:
You can't really provide it, it's just an option to set if you want a sdr unlock condition (default) or if you want to gift the storage deposit
But added a test for that